services/web/mgmt/2016-09-01/web/webapi/interfaces.go

```diff
@@ -0,0 +1,603 @@
+package webapi
```

Comment:
just to check is this the intended name for the package ? I thought it would just be `api`

Reply:
This is intentional so that all the `api` package names don't collide.